indx.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Abbas Aljuaifari</title>
+
+      <!--stylesheet-->
+
+    <link rel="stylesheet" href="style.css">
+
+     <!--google fonts-->
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer">
+
+</head>
+<body>
+
+    <!--Header with navbar starts here-->
+
+    <header class="header" id="header">
+        <a href="#"><img src="images/MyLogo.png"></a>
+
+        <!-- toggle button -->
+        <nav>
```

Comment:
- [ ] You did great trying to add the website logo. :taco: :smile:  But currently it seems your src link is wrong because the image doesn't display in the browser, I'll suggest you look into this. :+1: 
- SCREENSHOT:
![Selection_019](https://user-images.githubusercontent.com/107632780/235004693-52c992e8-90a8-49e8-b39e-9ade2c7047ed.png)



- [ ] I'll also suggest you add an `alt` attribute as it's amongst the best practice in web development and it also Strengthens Your SEO.

- [OPTIONAL] I noticed that it seems you made a typo error in your file name. It's currently `indx.html` instead of `index.html`. Although you could give your files any name, its the best practice to give them meaningfully and also avoid spelling errors and it makes your project appear professional. :wave: